modules/cassandra/build.gradle

```diff
@@ -0,0 +1,10 @@
+description = "TestContainers :: Cassandra"
+
+dependencies {
+    compile project(":database-commons"),
```

Comment:
Can you please add each dependency on its own line, with a `compile`? :slightly_smiling_face:

Reply:
Sure.